nsv13/code/game/gamemodes/pvp/pvp.dm

```diff
@@ -0,0 +1,237 @@
+/**
+
+Shamelessly ripped off from nuclear.dm
+
+*/
+
+GLOBAL_LIST_EMPTY(syndi_crew_spawns)
+
+GLOBAL_LIST_EMPTY(syndi_crew_leader_spawns)
+
+/datum/game_mode/pvp
+	name = "PVP"
+	config_tag = "pvp"
+	report_type = "nuclear"
+	false_report_weight = 10
+	required_players = 30 // 30 players initially, with 15 crewing the hammurabi and 15 crewing the larger, more powerful hammerhead
+	required_enemies = 10
+	recommended_enemies = 10
+	antag_flag = ROLE_SYNDI_CREW
+	enemy_minimum_age = 0
+
+	announce_span = "danger"
+	announce_text = "Nanotrasen's incursions into Syndicate space have not gone unnoticed!\n\
+	<span class='danger'>Operatives</span>: Eliminate Nanotrasen's presence in Syndicate space using the weaponry of the SSV Hammurabi, or via a thermonuclear detonation.\n\
+	<span class='notice'>Crew</span>: Survive the Syndicate assault long enough for reinforcements to arrive."
+
+	title_icon = "nukeops"
```

Comment:
Needs a actual icon

Reply:
When I get time. For now, it'll do. Theyre functionally similar enough anyway